src/main/java/com/softteam/vocabuilder/util/validations/Validations.java

```diff
@@ -1,17 +1,21 @@
 package com.softteam.vocabuilder.util.validations;
 
+import com.softteam.vocabuilder.exections.VocabuilderIllegalArgumentException;
 import lombok.Data;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
 
 import java.util.UUID;
 
 @Data
+@Component
 public class Validations {
     public UUID validateUUIDType(String parameter) {
         UUID uuidText = new UUID(0L,0L);
         try {
             uuidText = UUID.fromString(parameter);
         } catch (Exception e) {
-            throw new RuntimeException(e);
+            throw new VocabuilderIllegalArgumentException("validation - the id provided is not valid", HttpStatus.NOT_FOUND);
```

Comment:
That the uuid is not valid, it does not mean that the it was not "found" because you never searched for it

Reply:
there what I am trying to do is verify that the String passed has the UUID format, that it is in the DB or not, I do not validate it here.

Reply:
I understand, in that case the error is not `HttpStatus.NOT_FOUND` because it means, you searched and you didn't find it
the error when some input is not correct is `BAD_REQUEST`